ynr/apps/popolo/migrations/0022_popolate_person_image_from_image.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
```

Comment:
Should the name of this file be `populate_person_image_from_image`, or was the pun intentional? :D

Reply:
Ha! It's a nice pun, but I changed it